sdk/workflow_run.go

```diff
@@ -158,6 +158,15 @@ type WorkflowNodeJobRun struct {
 	SpawnInfos        []SpawnInfo `json:"spawninfos" db:"-"`
 }
 
+//WorkflowNodeJobRunInfo represents info on a job
+type WorkflowNodeJobRunInfo struct {
+	ID                   int64       `json:"id" db:"id"`
```

Comment:
the db annotation are useless because you don't use gorp to manage WorkflowNodeJobRunInfo